File: src/main/java/gg/ethereallabs/blockChess/utils/SyncHelper.java
package gg.ethereallabs.blockChess.utils;

import gg.ethereallabs.blockChess.BlockChess;
import org.bukkit.Bukkit;

public class SyncHelper {
    private SyncHelper() {}

    public static void runSync(Runnable block) {
        Bukkit.getScheduler().runTask(BlockChess.instance, block);
    }
}

File: bin/main/gg/ethereallabs/blockChess/utils/SyncHelper.kt
package gg.ethereallabs.blockChess.utils

import gg.ethereallabs.blockChess.BlockChess
import org.bukkit.Bukkit

object SyncHelper {
    fun runSync(block: () -> Unit) {
        Bukkit.getScheduler().runTask(BlockChess.instance, Runnable { block() })
    }
}